Source/dvlnet/base.cpp

```diff
@@ -470,7 +477,10 @@ bool base::SNetDropPlayer(int playerid, uint32_t flags)
 		LogError("make_packet: {}", pkt.error().what());
 		return false;
 	}
-	send(**pkt);
+	if (tl::expected<void, PacketError> result = send(**pkt); !result.has_value()) {

```

Comment:
Nit: personally I prefer this style
```suggestion
	tl::expected<void, PacketError> result = send(**pkt);
	if (!result.has_value()) {
```

Reply:
No objections here.